cmd/configure.go

```diff
@@ -102,11 +109,14 @@ var SSHKeysConfigCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(configureCmd)
-	configureCmd.PersistentFlags().BoolVar(&overwrite, "overwrite", false, "overwrite config files (default is `false`)")
+	configureCmd.PersistentFlags().BoolVar(&overwrite, "overwrite", false, "overwrite config files")
 	configureCmd.AddCommand(etcdConfigCmd)
 	configureCmd.AddCommand(masterConfigCmd)
 	configureCmd.AddCommand(NodeConfigureCmd)
 	configureCmd.AddCommand(openVPNConfigCmd)
 	configureCmd.AddCommand(openVPNClientConfigCmd)
 	configureCmd.AddCommand(SSHKeysConfigCmd)
+	openVPNClientConfigCmd.PersistentFlags().BoolVar(&revoke, "revoke", false, "revoke client certificate")
+	openVPNClientConfigCmd.PersistentFlags().StringVar(&clientName, "client-name", clientName, "The name of the user. It will be used as the CN of the client certificate")
+	openVPNClientConfigCmd.MarkFlagRequired("client-name")
```

Comment:
This means that the command now to revoke a user's certificate is `furyagent openvpn-client --revoke --client-name <client CN>` instead of using the list from the config file right?

Reply:
Exactly, I've removed OpenVPN client handling from furyagent's configuration file.